test/AssetPool.test.js

```diff
@@ -227,6 +227,29 @@ describe("AssetPool", () => {
         ).to.be.closeTo("45454545454545454545", assertionPrecision)
       })
     })
+
+    context(
+      "when collateral tokens sent to asset pool outside of cov pools",
+      () => {
+        beforeEach(async () => {
+          // Deposit some small amount of collateral tokens so that the amount
+          // of underwriter tokens is not 0
+          await assetPool.connect(underwriter1).deposit(1)
+          // Send collateral tokens to the asset pool outside cov pools
```

Comment:
Maybe:
```
// Send collateral tokens to the asset pool directly, without minting underwriter tokens
```

The second part is essential to understand why `toMint` is `0`.

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/145/commits/61952dd033344000712d1a74a06244b2f38331dc.